python/ray/data/tests/test_stats.py

```diff
@@ -290,13 +337,22 @@ def test_dataset_stats_basic(
                 f"* Output size bytes per block: N min, N max, N mean, N total\n"
                 f"* Output rows per task: N min, N max, N mean, N tasks used\n"
                 f"* Tasks per node: N min, N max, N mean; N nodes used\n"
-                f"{gen_extra_metrics_str(STANDARD_EXTRA_METRICS, verbose_stats_logs)}"  # noqa: E501
+                f"{gen_extra_metrics_str(STANDARD_EXTRA_METRICS_TASK_BACKPRESSURE, verbose_stats_logs)}"  # noqa: E501
             )
 
     for batch in ds.iter_batches():
         pass
     stats = canonicalize(ds.materialize().stats())
 
+    extra_metrics_1 = gen_extra_metrics_str(
```

Comment:
Are these the same / why do we need two extra metrics here?

Reply:
thanks, consolidated into one